compiler.events-only.toml

```
@@ -61,3 +61,36 @@ end = { date = "2024-09-27T17:00:00.00", timezone = "Europe/London" }
 status = "confirmed"
 organizer = { name = "davidtwco", email = "david@davidtw.co" }
 recurrence_rules = [ { frequency = "weekly" } ]
+
+[[events]]
+uid = "1731080669704"
+title = "steering: P-high Issue Review"
+description = "Continue going over our P-high tagged issues"
+location = "#t-compiler/meetings on Zulip (https://rust-lang.zulipchat.com/#narrow/stream/238009-t-compiler.2Fmeetings/)"
+last_modified_on = "2025-02-11T15:46:00.00Z"
+start = { date = "2024-11-22T10:00:00.00", timezone = "America/New_York" }
+end = { date = "2024-11-22T11:00:00.00", timezone = "America/New_York" }
+status = "confirmed"
+organizer = { name = "t-compiler", email = "compiler@rust-lang.org" }
+
+[[events]]
+uid = "1731080723243"
+title = "design: type_id is not sufficiently collision-resistant"
+description = "type_id is not sufficiently collision-resistant"
+location = "#t-compiler/meetings on Zulip (https://rust-lang.zulipchat.com/#narrow/stream/238009-t-compiler.2Fmeetings/)"
+last_modified_on = "2025-02-11T15:46:00.00Z"
+start = { date = "2024-12-06T10:00:00.00", timezone = "America/New_York" }
+end = { date = "2024-12-06T11:00:00.00", timezone = "America/New_York" }
+status = "confirmed"
+organizer = { name = "t-compiler", email = "compiler@rust-lang.org" }
+
+[[events]]
+uid = "1731080716169"
+title = "design: LLVM optional target-cpu features enabled by default"
+description = "LLVM optional target-cpu features enabled by default"
+location = "#t-compiler/meetings on Zulip (https://rust-lang.zulipchat.com/#narrow/stream/238009-t-compiler.2Fmeetings/)"
+last_modified_on = "2025-02-11T15:46:00.00Z"
+start = { date = "2024-12-13T10:00:00.00", timezone = "America/New_York" }
+end = { date = "2024-12-13T11:00:00.00", timezone = "America/New_York" }
```

I believe we were okay with sliding this forward 1 hour, relative to the other meetings?
```suggestion
start = { date = "2024-12-13T11:00:00.00", timezone = "America/New_York" }
end = { date = "2024-12-13T12:00:00.00", timezone = "America/New_York" }
```

---

yes, that was my intention but I got confused when translating the discussion on Zulip on actual TZ :) Thanks for the correction